src/libraries/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -101,17 +101,15 @@ namespace System.Reflection.Metadata
         public System.Reflection.Metadata.StringHandle Name { get { throw null; } }
         public System.Reflection.Metadata.BlobHandle PublicKey { get { throw null; } }
         public System.Version Version { get { throw null; } }
-#if !NETSTANDARD1_1
```

Comment:
I believe this is removed by GenerateReferenceSource tool, doesn't think we should remove it

Reply:
I _think_ we are currently removing these.

Reply:
I _believe_ we are currently removing these. I've merged PRs in the past that explicitly remove `netcoreapp1.x` artifacts.

Reply:
i see, JFYI there is build failures on these rows something like `The type or namespace name 'AssemblyName' does not exist in the namespace 'System.Reflection'`. Which makes me to think we might removed only netcoreapp1.x artifacts but not netstandard1.x, anyways i leave it for your preference

Reply:
I think you're right. The question then is why they're being removed by the generator. I'm hesitant to manually add those back, since that means we'd have to do this every time we update the signatures.

Reply:
That might be question for @safern